bot/modules/commands.js

```diff
@@ -3,6 +3,7 @@ let config = require("config");
 let miningChannel = config.get("Channels").mining;
 let randomChannel = config.get("Channels").random;
 let verificationChannel = config.get("Channels").verification;
+let bot-sandboxChannel = config.get("Channels").sandbox;
```

Comment:
please keep the styling a bit and change to for example botSandboxChannel

Reply:
fixed